api/osf_api.py

```diff
@@ -228,6 +228,16 @@ def get_providers_list(session=None, type='preprints'):
     return session.get(url)['data']
 
 
+def get_provider_status(provider):
+    """Return the boolean attribute `allow_submissions` from the dictionary object (provider)
+    """
```

Comment:
I can't think of a better name other than `get_provider_status()` although allowing submission is just one of the **_status_** of a provider. (`does_provider_allow_submission()` sounds worse ...) The DocStr for this function explains what it does so all good.

There is still an improvement but I am not sure: let `get_provider_status()` return a dictionary where `allow_submission` is just a key. Thus the name is more consistent with the implementation and we can add other status in the future.

Reply:
Let's name this `get_provider_submission_status`